src/clients/HubPoolClient.ts

```diff
@@ -270,6 +270,13 @@ export class HubPoolClient extends BaseAbstractClient {
     if (!isDefined(this.currentTime)) {
       throw new Error("HubPoolClient has not set a currentTime");
     }
+
+    if (deposit.quoteTimestamp > this.currentTime) {
+      throw new Error(
+        `Cannot compute lp fee percent for quote timestamp ${deposit.quoteTimestamp} in the future. Current time: ${this.currentTime}.`
+      );
+    }
```

Comment:
Should we also log here?

Reply:
Wouldn't the error show up in logging when it gets handled?

Reply:
True, fair point!